app/views/layouts/application.html.erb

```diff
@@ -20,6 +20,7 @@
   </head>
 
   <body>
+    <%= google_tag_manager %>
```

Comment:
I don't really know what Google Tag Manager is, or what it's doing, but should this be in the `body`?

Reply:
Yeah, according to the readme at https://github.com/rcs/google-tag-manager-rails